NOTICES.md

```diff
@@ -5,7 +5,9 @@ List of third-party dependencies grouped by their license type
 * Caffeine cache ([com.github.ben-manes.caffeine:caffeine:2.8.1](https://github.com/ben-manes/caffeine))
 * akka-actor-testkit-typed ([com.typesafe.akka:akka-actor-testkit-typed_2.13:2.6.10](https://akka.io/))
 * akka-discovery ([com.typesafe.akka:akka-discovery_2.13:2.6.10](https://akka.io/))
+* akka-http-jackson ([com.typesafe.akka:akka-http-jackson_2.13:10.1.11](https://akka.io))
 * akka-http-jackson ([com.typesafe.akka:akka-http-jackson_2.13:10.2.1](https://akka.io))
+* akka-http ([com.typesafe.akka:akka-http_2.13:10.1.11](https://akka.io))
 * akka-http ([com.typesafe.akka:akka-http_2.13:10.2.1](https://akka.io))
```

Comment:
Yep, FRS uses `10.2.x` because that was most recent at the time.
Then S3RS and S3DR use alpakka, which pulls in `10.1.x`.
When loading classpath jars, it's difficult to shade akka-http, so:
 a) FRS loads no classpath jars as a workaround
 b) DS must be compatible with readers
DS uses `10.1.x` to be compatible with S3DR

In an ideal world, you'd get the maven-shade working properly in readers, but we haven't done that yet...